src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -0,0 +1,425 @@
+---
+title: 'Advanced Data Scrubbing (Preview)'
```

Comment:
Do we want to replace "(Preview)" here with "Beta" as we discuss it as a beta option in the text that follows?

Reply:
done